src/GitUtils.ps1

```diff
@@ -405,3 +405,35 @@ function Update-AllBranches($Upstream = 'master', [switch]$Quiet) {
 
     git checkout -q $head
 }
+
+<#
+.SYNOPSIS
+    Gets a Git remotes object
+.DESCRIPTION
+    Gets a git remotes object
+    The object contains the name and URL of all remotes
+.EXAMPLE
+    PS C:\> $s = Get-GitRemotes
+    Returns:
+        Name   Url
+        ----   ---
+        origin git@github.com:dahlbyk/posh-git.git
+.INPUTS
+    None
+.OUTPUTS
+    System.Management.Automation.PSObject
+#>
+function Get-GitRemotes {
```

Comment:
PowerShell usually uses singular nouns e.g Stop-Process not Stop-Processes.  I would recommend changing this to `Get-GitRemote`.

Reply:
I totally agree on this as well that singular nouns should be used.